Source/src/core/rendering/Program.h

```diff
@@ -5,6 +5,29 @@ namespace Hachiko
 {
     class Program
     {
+    public:
+        enum class PROGRAMS
```

Comment:
Wow that's a cool thing, thanks for the improvement.

Reply:
Thx!